### PR TITLE
fix: use node env for mocha tests

### PR DESCRIPTION
**Pull request by barmac**
This was previously handled by recommended,
but it's not anymore due to node/browsert separation.

### CODE REVIEW

File: test/mocha/browser/browser.spec.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { expect } from 'chai';
+import { pick } from '../lib/object';
+
+const diagramXML = require('./diagram.bpmn');
+
+console.log(diagramXML);
```
Comment (marstamm) [inline test/mocha/browser/browser.spec.js:6]:
```suggestion
```

Comment (marstamm) [reply test/mocha/browser/browser.spec.js:6]: Or I might not understand how the tests work and this test that console.log is allowed?

Comment (barmac) [reply test/mocha/browser/browser.spec.js:6]: I added this on purpose to not trigger `no-unused-var`. The snippets in the tests are not executed but linted only.

Comment (marstamm) [reply test/mocha/browser/browser.spec.js:6]: I was to quick to post this review, makes sense to have this one here 👍

**Review by marstamm — APPROVED** [review]

File: test/mocha/browser/browser.spec.js (hunk 1 above)
Comment (marstamm) [reply test/mocha/browser/browser.spec.js:6]: Thank you for clarifying

**Pull request merged**